### ISSUE TITLE
panel transparency in dark mode not actually transparent

### ISSUE THREAD

**Issue by petemyron** [grafana/grafana#22582]
*Note*: I add a lot of style to my dashboards, and I stumbled on this after upgrading yesterday to 6.6.2 from 6.3.4ish. I've never seen anyone else do this with their dashboards, so maybe no one else is effected by it.

**What happened**:
In dark mode, creating a panel with 'transparent' toggled on, the panel background-color is set to one of the dark-mode colors (`dark-2` IIRC), but it's not actually transparent.

**What you expected to happen**:
I expected the background-color of the panel to be `rgb(0,0,0,0)` or `#00000000` (or anything with alpha 0).

**How to reproduce it (as minimally and precisely as possible)**:
1. dark mode
1. create dashboard
1. create panel (default `graph` panel type with transparent toggled on)
![image](https://user-images.githubusercontent.com/627937/75909644-5149f100-5e01-11ea-924d-44ac4acaa903.png)

1. set an actual background image/color/whatever. I do this like so:
![image](https://user-images.githubusercontent.com/627937/75909835-b0a80100-5e01-11ea-935c-736876ad3d79.png)

1. Notice the fill in the panels
![image](https://user-images.githubusercontent.com/627937/75910073-14cac500-5e02-11ea-8772-e6209a6ed56e.png)

**Anything else we need to know?**:
I set a background image like so:
1. enable `disable_sanitize_html = true` in grafana.ini
1. create a text panel with content pasted above
* it has to be above the fold due to lazy-loading not quite working correctly for text panels with more than just text (I have not tested in newer versions, btw)

Finally, I can work-around this issue by unsetting the background-color of the panel with css: `.panel-container--transparent { background-color: unset; }`

![image](https://user-images.githubusercontent.com/627937/75910476-c7028c80-5e02-11ea-83c3-87604dbefa31.png)

**Environment**:
- Grafana version: 6.6.2
- Data source type & version: n/a
- OS Grafana is installed on: docker on ubuntu on aws
- User OS & Browser: macOS Chrome
- Grafana plugins: every plugin (enterprise + variant on grafana-xxl)
- Others:

**Comment by torkelo** [comment]
transparent just means, same color as background here so no border & background color is same as background color

**Comment by berhanep** [comment]
You know what, I exactly had same issue with Transparent on/off with Grafana 7.x. I used the Boom Theme panel and defined my own background blue theme with css override, it used to work fine in Grafana version 6.2, the same dashboard, same panels except in Grafana 7.x it stopped working and was actually frustrating. If I fill my panel with background-color (255,255,255,0.2) for example toggling the transparent switch to 'on' doesn't make it transparent, it still shows the panel colour unfortunately. But now like you mentioned I added .panel-container--transparent { background-color: unset; } to my already existing (.panel background-colour) and all working as expected. REMEMBER, THIS DISCUSSION IS WHILE USING  THE BOOM THEME PLUGIN  and Transparent mode works in previous Grafana 6 versions but not in 7.x. What can I say, if I meet you in person I will buy you a pint mate.
Cheers.

PB
